src/lib/zf/doxygen/040_using.dox

```diff
@@ -113,9 +113,10 @@ production libraries:
   attribute, as described in the \ref attributes chapter.
 - As a convenience, the `-l` option to the `zf_debug` wrapper will set the
   \attrref{log_level} attribute to the specified value.
-- After the library initialization, the attributes attrref{log_level}, \attrref{log_format}
- attributes can be changed programmatically while the application is running using 
- \attrref{zf_attr_set_int}.
+- After library initialization, the \attrref{log_level} and \attrref{log_format}
+  attributes can be changed programmatically while the application is running using 
+  \attrref{zf_attr_set_int}.  Similarly, \attrref{log_file} can be modified
+  using \attrref{zf_attr_set_str}.
```

Comment:
I think it's worth adding a note that these attributes are global. e.g "\attrref{log_level} and \attrref{log_format} are global attributes and may be modified by anythread. Therefore, any users setting these levels from both threads must ensure thread safety."

Reply:
agreed. Thank you.